src/main/resources/docerina-templates/html/package.hbs

```diff
@@ -207,8 +214,39 @@
                         </div>
                         {{/each}}
                         <p></p>
-                        <br>{{/hasAnnotations}} 
-                    </div>
+                        <br>{{/hasAnnotations}}
+                        <!-- Enums -->
+                        {{#hasEnums}}
```

Comment:
I think it would be better if we place enums at the top, along with global variables. WDYT?

Reply:
Yeah, i'll move to right after global variables.

Reply:
Fixed with https://github.com/ballerinalang/docerina/pull/136/commits/5d18da2ffb6e5dae0128cb4458bfa37ae48bca3c